tests/cpp/t7477.nim

```diff
@@ -0,0 +1,11 @@
+{.emit: """/*TYPESECTION*/
```

Comment:
Move this to `tstaticimportcpp`.

You can use a type declaration such as `T data[N];` to make the code compatible with the older C++ standards.

Reply:
Done